catalog/sat-io/projects_sat-io_open-datasets_Harmonized_NTL_dmsp.jsonnet

```diff
@@ -0,0 +1,105 @@
+local id = 'projects/sat-io/open-datasets/Harmonized_NTL/dmsp';
+local subdir = 'sat-io';
+
+local ee_const = import 'earthengine_const.libsonnet';
+local ee = import 'earthengine.libsonnet';
+local spdx = import 'spdx.libsonnet';
+
+local license = spdx.cc_by_4_0;
+
+local basename = std.strReplace(id, '/', '_');
+local base_filename = basename + '.json';
+local self_ee_catalog_url = ee_const.ee_catalog_url + basename;
+
+{
+  // TODO: remove 'gee:skip_indexing'
+  'gee:skip_indexing': true,
+  'gee:user_uploaded': true,
+  stac_version: ee_const.stac_version,
+  type: ee_const.stac_type.collection,
+  stac_extensions: [
+    ee_const.ext_eo,
+    ee_const.ext_sci,
+  ],
+  id: id,
+  title: 'Harmonized Global Night Time Lights DMSP',
+  'gee:type': ee_const.gee_type.image_collection,
+  description: |||
+    This dataset comprises nighttime light (NTL) data sourced from the Defense Meteorological Satellite Program (DMSP)/Operational Linescan System (OLS) and the Visible Infrared Imaging Radiometer Suite (VIIRS) on the Suomi National Polar-orbiting Partnership satellite, offering comprehensive insights into human activities on both regional and global scales. While prior records of nocturnal illumination from DMSP (1992–2013) and VIIRS (2012–2018) exist, the historical NTL archive has remained underutilized due to substantial disparities between DMSP and VIIRS data. In this research endeavor, we have developed a consolidated and uniform NTL dataset at a global scope by harmonizing inter-calibrated NTL observations from DMSP and simulated DMSP-like NTL observations from VIIRS. This newly generated DMSP NTL time-series dataset, spanning from 1992 to 2018, exhibits coherent temporal patterns, presenting a valuable resource for diverse studies related to human activities, including electricity consumption and urban expansion dynamics.
+  |||,
+  license: license.id,
+  links: ee.standardLinks(subdir, id),
+  keywords: [
+    'nighttime_light',
+    'calibration',
+    'viirs',
+    'dmsp',
+  ],
+  providers: [
+    ee.producer_provider('Nature Scientific Data', 'https://www.nature.com'),
+    ee.host_provider(self_ee_catalog_url),
+  ],
+  extent: ee.extent_global('1992-01-01T00:00:00Z', '2013-12-31T00:00:00Z'),
+  summaries: {
+    'gee:schema': [
+      {
+        name: 'country',
+        description: 'Country',
+        type: ee_const.var_type.string,
+      },
+      {
+        name: 'year',
+        description: 'Year',
+        type: ee_const.var_type.double,
+      },
+    ],
+    gsd: [
+      1000,
+    ],
+    'eo:bands': [
+      {
+        name: 'b1',
+        description: 'DN',
```

Comment:
Description is too short; what is this, and in what units?

Reply:
Removed NTL completely but DN means Digital Number it is well known in remote sensing community

Reply:
Ack.